things/model/src/main/java/org/eclipse/ditto/things/model/signals/events/ThingEventToThingConverter.java

```diff
@@ -121,6 +121,17 @@ private static Map<Class<?>, BiFunction<ThingEvent<?>, ThingBuilder.FromScratch,
                     return ThingsModelFactory.newThing(mergedJson.asObject());
                 }
         );
+        mappers.put(ThingMigrated.class,
+                (te, tb) -> {
```

Comment:
This shouldn't just be copy&paste from `ThingMerged` - ThingMigrated e.g. always contains the whole migrated thing - so using a `resourcePath` does not make sense.